tardis/io/model/parse_density_configuration.py

```diff
@@ -0,0 +1,201 @@
+import numpy as np
+from astropy import units as u
+
+from tardis.io.config_reader import Configuration
+from tardis.util.base import quantity_linspace
+
+
+def parse_density_section(
+    density_configuration: Configuration,
+    v_middle: u.Quantity,
+    time_explosion: u.Quantity,
+):
+    """
+    Parse the density section of the configuration file and produce a density at
+    time_explosion.
+
+    Parameters
+    ----------
+
+    density_configuration : tardis.io.config_reader.Configuration
+    v_middle : astropy.Quantity
+        middle of the velocity bins
+    time_explosion : astropy.Quantity
+        time of the explosion
```

Comment:
Docstring should describe type and structure of objects returned.

Reply:
I've added the type hints now but can also add a description